docs/src/main/asciidoc/security-basic-authentication-tutorial.adoc

```diff
@@ -537,8 +538,8 @@ public class Role extends PanacheEntity {
 By default, passwords are stored and hashed by using https://en.wikipedia.org/wiki/Bcrypt[bcrypt] under the 
 https://en.wikipedia.org/wiki/Crypt_(C)[Modular Crypt Format] (MCF).
 
-When you need to create a hashed password we provide the convenient `String BcryptUtil.bcryptHash(String password)`
-function, which defaults to creating a random salt and hashing in 10 iterations. 
+When you need to create a hashed password, provide the convenient `String BcryptUtil.bcryptHash(String password)`
```

Comment:
No, the "we" means Quarkus - the user doesn't have to provide it. 
The original meaning was correct, maybe to improve I would change "we provide" to "you may use"

Reply:
OK, applying